### PR TITLE
Update terraform providers

### PR DESCRIPTION
**Pull request by ipochi**

### CODE REVIEW

File: assets/terraform-modules/bootkube/versions.tf [hunk 1]
```diff
@@ -5,7 +5,7 @@ terraform {
   required_providers {
```
Comment (invidian) [inline assets/terraform-modules/bootkube/versions.tf:5]: Why is there `10` in the commit message?

File: pkg/assets/generated_assets.go [hunk 2]
```diff
@@ -5359,9 +5359,9 @@ var vfsgenAssets = func() http.FileSystem {
 		"/terraform-modules/aws/flatcar-linux/kubernetes/versions.tf": &vfsgen۰CompressedFileInfo{
```
Comment (invidian) [inline pkg/assets/generated_assets.go:5359]: To be honest, I'm not a fan of committing assets generation separately, as it breaks the atomicity of the commits. Maybe we can just commit everything as one commit?

**Review by invidian — DISMISSED** [review]
Looks good

**Review by invidian — APPROVED** [review]
:heart:

**Review by surajssd — APPROVED** [review]
Quick LGTM :laughing:

**Pull request merged**